uniffi_bindgen/src/bindings/python/gen_python.rs

```diff
@@ -217,6 +221,8 @@ mod filters {
                 nm,
                 class_name_py(&type_.canonical_name())?
             ),
+            Type::Wrapped { prim, .. } => lift_py(nm, prim.as_ref())?,
+            Type::External { .. } => panic!("should not be necessary to lift External types"),
```

Comment:
Maybe these should be implemented for `Type::External`?  It seems like you would need that if you had a function that returned and/or inputted an external type directly (not as a field in a record like the current code).